### PR TITLE
Make plot aesthetics equal to matlab 

### PR DESCRIPTION
**Pull request by guillermoaguilar**
For two plots:
- plot psychometric function
- plot marginal

### CODE REVIEW

File: psignifit/psigniplot.py [hunk 1]
```diff
@@ -213,10 +218,15 @@ def plot_prior(result: Result,
                line_color: Union[str, List[float], np.ndarray] = '#0069AA',  # blue
                line_width: float = 2,
                marker_size: float = 30):
-    """ Plot the priors on the different parameters.
-
-    The coloured psychometric functions correspond to the 0%, 25%, 75% and 100%
-    quantiles of the prior.
+    """ Plot the priors for the threshold, width and lambda parameters.
+    
+    The upper pannels show the priors. The lower pannels show a set of psychometric
```
Comment (pberkes) [inline psignifit/psigniplot.py:223]: "pannels" -> "panels" (3x)
(a pannel is apparently the stomach of a hawk)

File: psignifit/psigniplot.py [hunk 2]
```diff
@@ -168,8 +173,8 @@ def plot_marginal(result: Result,
     """ Plots the marginal for a single dimension.
 
     Args:
-        result: should be a result struct from the main psignifit routine
-        dim: The parameter to plot. 'threshold', 'width', 'lambda', 'gamma', 'eta'
+        result: should be a result objeect from the main psignifit routine
```
Comment (pberkes) [inline psignifit/psigniplot.py:176]: type: "object"

**Review by pberkes — APPROVED** [review]
LGTM, a couple of typos to fix

File: psignifit/psigniplot.py (hunk 2 above)
Comment (otizonaizit) [reply psignifit/psigniplot.py]: fixed

File: psignifit/psigniplot.py (hunk 1 above)
Comment (otizonaizit) [reply psignifit/psigniplot.py]: fixed

**Comment by otizonaizit** [comment]
I fixed the typos. Ready to merge.

**Comment by guillermoaguilar** [comment]
thank you for detecting and fixing :)

**Pull request merged**